addon/components/environment-banner.hbs

```diff
@@ -1,12 +1,22 @@
-<AuAlert @size='tiny' @skin='warning' class='au-u-margin-bottom-none'>
-  <div class='au-u-flex au-u-flex--spaced au-u-flex--between'>
-    <p class='au-u-medium'>
-      Environment:
-      {{@environmentName}}
+<AuAlert @size='tiny' @skin={{this.environmentValues.skin}} class='au-u-margin-bottom-none'>
+  <div class='au-u-flex au-u-flex--column au-u-flex--spaced-tiny au-u-flex--start'>
+    <p class='au-u-flex au-u-medium au-u-1-5@medium au-u-flex--vertical-center'>
+      Environment: {{this.environmentValues.env}}
     </p>
-    <div class='au-u-flex au-u-flex--spaced-small'>
-      <p>
-        {{@applicationName}}
+    <div class='au-u-flex au-u-flex--column au-u-flex--spaced-tiny au-u-4-5@medium au-u-flex--between'>
+      <p class='au-u-4-5@medium'>
+        Dit is de <strong>{{this.environmentValues.title}}</strong> van het {{this.applicationName}} met fictieve en testgegevens.
+        De productieomgeving met de echte data vind je op 
+        <a 
+          rel="tag noopener noreferrer" target="_blank"
+          title={{this.applicationName}}
+          href={{@applicationUrl}}
+        >
+          {{@applicationUrl}}
+        </a>.
```

Comment:
I think here you're making too many assumptions. Other applications using the banner might not want to have text in the middle, or might want something else written here. We have to keep this addon as generic as possible. 
To keep it generic while still being able to write things in here, you could consider the [block content](https://guides.emberjs.com/release/components/block-content/) approach in the component.